src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -411,6 +411,12 @@
     <Compile Include="..\..\Common\src\Interop\Windows\ole32\Interop.CoTaskMemAllocFree.cs">
       <Link>Interop\Windows\ole32\Interop.CoTaskMemAllocFree.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\Interop\Windows\ole32\Interop.CoInitializeEx.cs">
```

Comment:
It looks like everything that needs these is in CoreLib. What's the reason for putting them under Common?

Reply:
The convention that we have inherited from CoreFX is to put all Interop decls under Common. I am following the pattern for consistency. I agree that it does not feel always right.